contract-addresses.json

```diff
@@ -0,0 +1,4 @@
+{
+  "Vault": "0x20c2554429dD269FB7785a1FCA08a04a70484E5a",
```

Comment:
In the current structure of this repository, these addresses should live in `protocol-data.json`.  These two addresses are from version `v0.11.0` (and `v0.11.1`) of the `@perp/lushan` npm package.

We should nail down whether we will use the `@perp/lushan` npm package to directly get these addresses and what version to use if we use it.  If we do not use it, we should determine how to extract and store the appropriate contract/account addresses within the repository.